fusion_docs/guide/local.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: Nextflow CLI local execution
```

Comment:
I would just say 'Local execution' as the CLI can be used for other types of execution as well